### PR TITLE
Add missing doc & news for PR 1220

### PR DESCRIPTION
**Pull request by jasl**
NOTE: changes in gemfiles is made by Appraisal

### CODE REVIEW

File: gemfiles/rails_5_2.gemfile [hunk 1]
```diff
@@ -2,15 +2,16 @@
...
-gem "rails", "~> 5.2.2", ">= 5.2.2.1"
+gem "rails", "~> 5.2.0"
 gem "rspec-core", git: "https://github.com/rspec/rspec-core.git"
 gem "rspec-expectations", git: "https://github.com/rspec/rspec-expectations.git"
 gem "rspec-mocks", git: "https://github.com/rspec/rspec-mocks.git"
 gem "rspec-rails", branch: "4-0-dev", git: "https://github.com/rspec/rspec-rails.git"
 gem "rspec-support", git: "https://github.com/rspec/rspec-support.git"
-gem "sqlite3", "~> 1.3", "< 1.4", platform: %i[ruby mswin mingw x64_mingw]
-gem "tzinfo-data", platforms: %i[mingw mswin x64_mingw]
+gem "rubocop", "~> 0.66"
```
Comment (nbulaj) [inline gemfiles/rails_5_2.gemfile]: Rubocop produces warnings for non-alphabetical sorted gems AFAIR

File: NEWS.md [hunk 2]
```diff
@@ -7,6 +7,7 @@ User-visible changes worth mentioning.
 
 ## master
 
+- [#1220] Drop Rails 4.2 & Ruby < 2.4 support.
```
Comment (nbulaj) [inline NEWS.md]: I already added it to master :)

File: gemfiles/rails_5_2.gemfile (hunk 1 above)
Comment (jasl) [reply gemfiles/rails_5_2.gemfile]: I think we should let Rubocop ignores this dir

Comment (nbulaj) [reply gemfiles/rails_5_2.gemfile]: Ah, it's Appraisal. Make sense. Could you please add it?

Comment (jasl) [reply gemfiles/rails_5_2.gemfile]: I already done this <https://github.com/doorkeeper-gem/doorkeeper/blob/master/.rubocop.yml#L7>

**Comment by jasl** [comment]
Indeed :smile:

**Pull request merged**